client/extensions/zoninator/components/settings/zones-dashboard/index.jsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import page from 'page';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { find, flowRight, get, trimEnd } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CompactCard from 'components/card/compact';
+import HeaderCake from 'components/header-cake';
+import SectionHeader from 'components/section-header';
+import sectionsModule from 'sections';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+const getZones = () => {
+	return [
+		{ label: 'Foo', slug: 'foo', description: 'My first zone' },
+		{ label: 'Bar', slug: 'bar', description: 'Another zone' },
+		{ label: 'Baz', slug: 'baz', description: 'Another zone' },
+		{ label: 'Boo', slug: 'boo', description: 'Another zone' },
+	];
+};
```

Comment:
Should this be a function? I imagine that later, this kind of data will be returned by selector, so it's likely to become an (array) prop -- not a function...